dev/maven-init.gradle

```diff
@@ -0,0 +1,7 @@
+allprojects {
```

Comment:
If this file accepts comments, you might want to explain it

Reply:
Done.